mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.cpp

```diff
@@ -158,29 +171,50 @@ MPFaceMeshDetector::DetectFacesWithStatus(const cv::Mat &camera_frame,
         (height >> 1);
     bounding_box.width = width;
     bounding_box.height = height;
-  }
+}
+
+mediapipe::Packet poses_packet;
+if (!poses_poller_ptr) {
+    return absl::CancelledError(
+        "Failed during getting next poses_packet. 1");
+}
 
-  // Get face landmarks.
-  if (!landmarks_poller_ptr ||
-      !landmarks_poller_ptr->Next(&face_landmarks_packet)) {
+if (!poses_poller_ptr->Next(&poses_packet)) {
+    return absl::CancelledError(
+        "Failed during getting next poses_packet. 2");
+}
+
+auto& face_poses = poses_packet.Get<::std::vector<Eigen::Matrix4f>>();
```

Comment:
It would be good to move the part with getting the pose to the separate method the same way as it was done for landmarks.